generator/Makefile

```diff
@@ -399,3 +400,10 @@ $(MIBDIR)/readydataos:
 $(MIBDIR)/powercom:
 	@echo ">> Downloading powercom"
 	@curl $(CURL_OPTS) -o $(MIBDIR)/powercom "$(POWERCOM_URL)"
+
+$(MIBDIR)/cisco_imc:
+	$(eval TMP := $(shell mktemp))
+	$(eval TMP_DIR := $(shell mktemp -d))
+	@echo ">> Downloading Cisco UCS to $(TMP)"
+	@curl $(CURL_OPTS) $(CURL_USER_AGENT) -o $(TMP) $(CISCO_CUCS_URL)
+	@tar -xf $(TMP) -C $(MIBDIR)/cisco_imc
```

Comment:
this dir `$(MIBDIR)/cisco_imc` does not exist, we'd either need to create it or even better just extract the two used mibfiles (`CISCO-UNIFIED-COMPUTING-FAULT-MIB.my` `CISCO-UNIFIED-COMPUTING-STORAGE-MIB.my`)

Reply:
I see that CISCO-UNIFIED-COMPUTING-FAULT-MIB.my CISCO-UNIFIED-COMPUTING-STORAGE-MIB.my, they have many dependencies like CISCO-UNIFIED-COMPUTING-TC-MIB; CISCO-UNIFIED-COMPUTING-MIB CISCO-SMI CISCO-TC
that I know I can found that in the tar file.
What do you think I should do?

Reply:
fixed that with line 408, wdyt? @bastischubert